agent/config/runtime.go

```diff
@@ -1686,6 +1687,55 @@ func (c *RuntimeConfig) ClientAddress() (unixAddr, httpAddr, httpsAddr string) {
 	return
 }
 
+func (c *RuntimeConfig) ConnectCAConfiguration() (*structs.CAConfiguration, error) {
+	if !c.ConnectEnabled {
+		return nil, nil
+	}
+
+	ca := &structs.CAConfiguration{
```

Comment:
Why not continue to get these default values from [here](https://github.com/hashicorp/consul/blob/master/agent/consul/config.go#L594)?

Reply:
We cannot import that package as it would be a cyclical dependency.

Reply:
Although we could move the defaults into consts in the structs package?

Reply:
I moved the constants into the structs package and reference them from both places.